hooks/useCustomStyle.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useState } from "react";
+
+function isMobile() {
+    return globalThis.matchMedia("(max-width: 767px)").matches
+}
+
+function getOneSidePadding(mainElement: HTMLElement) {
+
+    return Number(getComputedStyle(mainElement).paddingLeft.replace("px", "")) ;
+}
+
+function getPseudoElementWidth(currentElement: HTMLElement, pseudoName : string) {
+
+    return Number(getComputedStyle(currentElement, pseudoName).width.replace('px', ''))
+}
+
+function isInCodeContainer(currentElement : HTMLElement) {
+    const containerElement = currentElement.closest('.code-container')!
+    return containerElement?.scrollLeft
+}
+
+function getValuesForCSSLeft( currentElement : HTMLElement) {
+
+    const screenWidth = globalThis.window.outerWidth;
```

Comment:
`window` is normally accessed directly. 
Also, iirc, `window` does not have any border width or padding, which is why `window.innerWidth` is more commonly used.

Reply:
opinion about isMobile <- applied
prettier  <- applied
window.innerWidth <- applied